admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
----
-showToc: false
----
-<swagger>https://raw.githubusercontent.com/mojaloop/central-ledger/master/src/api/interface/swagger.json</swagger>
\ No newline at end of file
+---
+showToc: true
+---
+# Introduction
+
+This document provides detailed information about the Central Ledger API. The Central Ledger API is a Mojaloop API enabling Hub Operators to manage admin processes around:
+
+- creating/activating/deactivating participants in the Hub
+- adding and updating participant endpoint information
+- managing participant accounts, limits, and positions
+- creating Hub accounts
+- performing Funds In and Funds Out operations
+- creating/updating/viewing settlement models
+- retrieving transfer details
+
+ For background information about the participant and settlement model details that the Hub Operator can administer using the Central Ledger API, see section [Basic concepts](#basic-concepts).
+
+## Basic concepts
+
+To provide context for the admin operations that the Central Ledger API enables, this section gives a brief definition of some basic concepts.
+
+**Participant**
+Either the Hub itself or a Digital Financial Service Provider (DFSP) that is a participant in a Mojaloop scheme.

```

Comment:
nit: This section is a little hard to read with the formatting - maybe you can add make the heading and content on a different line? Or perhaps use `### Participant` instead of `**Participant**`

Reply:
OK, will make the change.